docs/grammar.txt

```diff
@@ -100,6 +105,8 @@ Identifier = [a-zA-Z_] [a-zA-Z_0-9]*
 HexNumber = '0x' [0-9a-fA-F]+
 DecimalNumber = [0-9]+
 
+ElementaryTypeNameExpression = ElementaryTypeName '(' expression ')'
```

Comment:
The `ElementaryTypeNameExpression` is **only** the `ElementaryTypeName`, it is not the cast. By the means of `ElementaryTypeNameExpression`, the `ElementaryTypeName` is brought into the realms of expressions.